COMMIT-GUIDELINES.md

```diff
@@ -0,0 +1,161 @@
+
+The two relevant prefixes are `fix` and `feat`. Commits with these prefixes should have clean messages about what they are fixing or what feature is being added, respectively. Each commit specifies the weight and impact of the commit on the project.
+
+Commits prefixed with `fix` are considered bug fixes, and so they are deemed to have little impact on the project, are backwards compatible with the previous version, and aren't going to cause any breaking changes. These commits trigger a patch version to be generated, which is version increment of `0.0.1` in the semantic versioning scheme.
+
+Commits prefixed with `feat` are considered features, which introduce new functionality into the project, such as a new component, new style change, or other significant change. These commits are backwards compatible as well and trigger a version increment of `0.1.0`.
+
+Not every commit needs to start with `fix` or `feat` within a branch, only the ones that have changes worth calling out in our `CHANGELOG.md` file. Commits can be prefixed with other prefixes: `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`. These are more suited for smaller parts of a larger update or commits that are being made that should not impact the version number, example project documentation updates.
+
+The changelog can be found within the [CHANGELOG.md](https://github.com/ongov/ontario-design-system/blob/develop/CHANGELOG.md) file, which a running history of all changes made in each release. The changelog version entries look like the following example:
+
+```
+# 5.0.0-alpha.3 (2024-12-17)
+
+### Bug Fixes
+
+- **grid.mixins:** declare $modules before use 1c7791f
+- **utils:** isNumber function validation inverted 37637b6
+- **utils:** update `retrieveEnumKeys` function 325d3af
+```
+
+#### Breaking changes
+
+A breaking change is a change that is significant enough to cause an incompatibility with the current state of the packages. This could be a removed component property, a set of styles that work differently, or a hole host of different changes.
+
+Breaking changes are denoted in the footer of the commit by starting it with `BREAKING CHANGE:` followed by a message about what the breaking change impacts.
+
+It is highly recommended to add a clear reason for the breaking change and what actions need to be taken when upgrading to the newer version. See the example under [Breaking change](#breaking-change) for an idea of how to communicate the change.
+
+It is important to note: Breaking changes can be added to any commit irregardless of the commit prefix, they are not just relegated to `fix` or `feat` commit prefixes as those aren't always relevant when introducing a breaking change.
```

Comment:
Consider bolding `It is important to note:`

Reply:
TIL! This is one of the contexts I use _irregardless_ to emphasis the lack of regard that needs to be taken, but I've updated it nonetheless.